### PR TITLE
[Community tutorial][edit existing] crop classification patch 

### PR DESCRIPTION
**Pull request by jdbcode**
Summary of changes:

- Max line length 80 characters
- Remove ### section formatting for subsections (a., b., c., etc) so that whole paragraphs are not bold.
- Add new line between section and paragraphs
- Use periods at the end of code comments
- Use "import" instead of "ingest" in code comments when loading a dataset
- Import 'users/juliusadewopo/Mzetrgt_*' within the code, instead of relying on the user to import
- Exponential notation for really long values (maxPixels)
- Ensure camelCase variable names
- Used key:value dictionary for function input so users know what each arg is.
- Add "cart" to variable names where CART was used
- Make visualization param dict for displaying the classificed map layers
- Make more distinctive name for the classified map layers (added "CART" and "RF")
- Clip the classified maps for display instead of overlaying the AOI
- Make the Nigeria boundary a line when displayed to the map
- Use Satellite as the basemap for easier interpretation/ better context
- Self-evident color scheme for the classes
- Use "validation" instead of "test" when referring to validation related variables
- Use cloud probability layer instead of QA band to mask clouds
- Use medoid instead of min composite
- Include the field points in the repo as csv files in case personal asset source is ever changed
- Clang Google code formatting
- Shorten title so that it will fit in the EE Developer Guide side nav bar

Script for testing code blocks:

https://code.earthengine.google.com/e5d09bf7b6378b352874b71cf51b503c

### CODE REVIEW

**Comment by jdbcode** [comment]
Hi @PJNation - please let me know if there is anything I can do to help you get started on the review of this PR.
Best regards, Justin.

**Comment by gino-m** [comment]
@PJNation looks like you @mentioned me in a commit in your fork:

> @gino-m : I think the changes look good so far. I may have to re-run the code to confirm results for section 4

@jdbcode is providing edits in the PR. Could you kindly click "Add your review" at the top right of this page and either Request changes or Approve? Thanks!

image

File: tutorials/classify-maizeland-ng/validation-pts.csv [hunk 1]
```diff
@@ -0,0 +1,576 @@
+CRP,Lat,Lon,Other_Crop,RSP,Yield_t_ha,class,id,otherc
```
Comment (gino-m) [inline tutorials/classify-maizeland-ng/validation-pts.csv:1]: Can we remove these files now that they're shared via Assets?

Comment (jdbcode) [reply tutorials/classify-maizeland-ng/validation-pts.csv]: Done

**Pull request merged**